### ISSUE TITLE
What is your stock radio running? It should have "radio info" in the menus.

### ISSUE THREAD

**Issue by P49K** [rogerclarkmelbourne/Radioddity_GD-77#36]
This is what's displayed.  

What is your stock radio running? It should have "radio info" in the menus.

_Originally posted by @forkoz in https://github.com/rogerclarkmelbourne/Radioddity_GD-77/issues/14#issuecomment-480017334_

This is what I get under "Radio info, Dev info"
Radio
BF-1801

s/n

CPS-VERSION

H-VERSION

SOFT-VERSION
V2.00.09

DSP-VERSION
HRC6000 V1.2

If "SOFT-VERSION" is the firmware then what is offered here is newer.
The question I would then have is what are the improvements and has
anyone tried it yet to determine it to be safe? 

Thanks,
Ken

**Comment by forkoz** [comment]
I can't determine it "safe" because I don't have an 1801 but it looks like the correct firmware and is newer than what you're running.

**Comment by P49K** [comment]
Its interesting the firmware is not available on the Baofeng site that I got my CPS from.
The dealer I purchased the radio from who also has the CPS told me they are not aware
of any firmware update.   
Ken

**Comment by forkoz** [comment]
Strings in the FW done at compile time indicate its for the 1801. You can check yourself. Whether it fixes anything or makes a feature worse on your end I can't attest to. GD-77 and MD-760 are the same radio with different available firmware too.

**Comment by P49K** [comment]
Without the original firmware to go back to I hate to take a chance.  
You guys have a talent to recover from a mistake like that.
I'll hang in there until more info becomes available.
Thanks, Ken

**Comment by forkoz** [comment]
There is no backup of the orignal so really no way to recover. I'm pretty sure your radio wouldn't break but if there is some new bug you would be stuck with it.

**Comment by P49K** [comment]
Maybe it could have something to do with locking out frequencies outside the Ham Bands.
Ken

**Comment by forkoz** [comment]
Doubt it, and that can be overriden anyway. Wish they put a change log.

**Comment by Miklor** [comment]
Radio
- BF-1801
S/N
CPS-Version
H-Version
Software Ver
- v2.01.09
DSP
- HRC6000 v1.02

**Comment by forkoz** [comment]
Ok, my radio came with 2.0.9 and I flashed to 2.1.9 no obvious problem.

**Comment by P49K** [comment]
anything new with 2.1.9 ?
Ken

**Comment by forkoz** [comment]
Yes, they added "group match" and "p2p match" which aren't in the manual. And I added DMRid support which has been working nicely.

**Comment by P49K** [comment]
Still new here to DMR,...
Group Match, would that be for RxGroup meaning I no longer have to create a Rx Group List?
p2p match, could you tell me what that means for this radio.
DMR id support, that sounds good but its something you modded in the firmware?
DMR id support saves me time from adding hundreds of contacts for the call sign to show?

Thanks,  Ken

**Comment by P49K** [comment]
Does the DMR id show call sign and name?
Ken

**Comment by forkoz** [comment]
Your guess is as good as mine. I tried to turn them on but so far notice no difference. Its not in the manual.

DMRid can show the callsign or name in the latest "expanded version" but you don't get both. And your contacts are free for other stuff. Works just like the GD77.

**Comment by P49K** [comment]
When you say expanded version,  is this a modification to the firmware
you have made?

Ken




____________________________________________________________
World In Shock By What Tiger's Daughter Looks Like Now
dailysovereign.com
http://thirdpartyoffers.netzero.net/TGL3231/5cb5fd8e9a3407d8e43ccst01duc

**Comment by forkoz** [comment]
Yes, I've ported most of the patches done to the GD77 and added a few others. Its all in the patches thread.
